lib/ohai/mixin/ec2_metadata.rb

```diff
@@ -200,6 +201,13 @@ def fetch_userdata
         response.code == "200" ? response.body : nil
       end
 
+      def fetch_dynamic_data
```

Comment:
I don't think it should be memoized. It's a dynamic data and some values from it can change any time.